php-git-hooks.yml.sample

```diff
@@ -23,6 +23,7 @@ pre-commit:
     phpcs:
         enabled:     true
         standard:    PSR2
+        ignore:      '*/migrations/*'
```

Comment:
When tool run from "composer install", don't ask about ignore option. It must ask.